test/common/utils.js

```diff
@@ -38,7 +38,7 @@ export function chooseRandomPanFromList(panList) {
 }
 
 export function getFCList() {
-    return papaparse.parse(open('../../assets/<FILENAME>.csv'), { header: true }).data;
+    return papaparse.parse(open('../../assets/fc_pgpans.csv'), { header: true }).data;
```

Comment:
@GiovanaSolorzano the committed `fc_pgpans.csv` is too large.
Can we keep the CSV file path parameterized and share the file by other means?
E.g. handle it as [secure file](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/download-secure-file-v1?view=azure-pipelines) in Azure DevOps.

Reply:
@TommasoLencioni The required behavior will be implemented later within this sprint. At the moment the file has been left directly in the repository to facilitate testing by other teams.